.gitignore

```diff
@@ -256,3 +256,6 @@ $RECYCLE.BIN/
 
 # Kopernicus
 build/GameData/Kopernicus/Cache/
+src/Kopernicus/Kopernicus.csproj
```

Comment:
This is not how git(ignore) works. You cannot tell it to ignore changes to a file, you can only tell it to not track the file at all. Since these files are already commited, adding them to gitignore won't change anything.

You would need to unstage the changes to these files in your client before making a commit (or if using the commandline, don't add them in the first place).

Reply:
The advice is appreciated as I'm new to git. I like it, but we use subversion at work (ick) and it's obviously a different beast. As for gitignore, I'm not too worried about it, just don't merge csproj changes if you don't want them.

Reply:
By the way, adding the file to .gitignore will *delete* the file off the repository because it is no longer tracked locally. The proper way to ignore changes to a file is by using either `git update-index --assume-unchanged <file>` or `git update-index --skip-worktree <file>`, where the former will only ignore changes until it is changed in the upstream (github) repository.

Reply:
Gotcha.  I'll revert the .gitignore changes and handle this better in my local repo.